EternoSeeker/gameoflife
# Add presets such as Spaceship  and Boat-tie

Title: Added middleweight spaceship and boat-tie presets in Conway's Game of Life

## Description
This pull request adds two new presets, the middleweight spaceship and boat-tie, to Conway's Game of Life. These presets are popular patterns in the Game of Life community and add more variety to the available options for users.

## Related Issue
Close #249 

## Motivation and Context
The addition of these presets enriches the experience for users by providing them with more interesting patterns to observe and interact with in Conway's Game of Life.

## How Has This Been Tested?
These presets have been tested locally to ensure they function correctly within the Game of Life simulation. Instructions for testing:
1. Run the Game of Life simulation.
2. Select the middleweight spaceship or boat-tie preset from the preset menu.
3. Observe the behavior of the pattern within the simulation.

## Checklist

## Screenshots (if necessary):

### Boat-Tie
![image](https://github.com/EternoSeeker/gameoflife/assets/111881649/edc5334a-a198-4fc4-99c4-41776e8be077)

### Spaceship
![image](https://github.com/EternoSeeker/gameoflife/assets/111881649/75d7f683-7e67-4c73-a2e0-1595faaf54d1)
